main.go

```diff
@@ -192,6 +193,11 @@ func runAudit(c conf.Configuration, auditPath string, outputFile string, outputU
 		panic(err)
 	}
 
+	if setExitCode && auditData.ClusterSummary.Results.Totals.Errors > 0 {
+		logrus.Errorf("Error found. Exiting audit.")
+		os.Exit(2)
```

Comment:
Did some research - apparently `2` [has special meaning](http://www.tldp.org/LDP/abs/html/exitcodes.html)

Can you change to `3`?